src/main/java/com/quantumlytangled/gravekeeper/GraveKeeperConfig.java

```diff
@@ -55,12 +57,14 @@
       "cofhcore:soulbound",
       "aoa3:intervention"
   };
-  public static String[] SOULBOUND_TAG_BOOLEAN = new String[]{
-      "Botania_keepIvy"
+  public static String[] SOULBOUND_TAG_VALUE = new String[]{
+      "Botania_keepIvy:1",
+      "spectreAnchor:0"
   };
   public static int KEEP_SOULBOUND_AMOUNT = 5;
 
   public static List<Enchantment> SOULBOUND_ENCHANTMENTS = null;
+  public static Map<String, Boolean> SOULBOUND_TAGS = null;
```

Comment:
SOULBOUND_TAG_VALUE & SOULBOUND_TAGS  are interchangeable. Consider using more specific names, for example
SOULBOUND_TAG_VALUE_STRINGS & SOULBOUND_TAG_VALUES_MAP or something alike?